packages/plugin-doc-coverage/src/lib/utils.unit.test.ts

```diff
@@ -0,0 +1,95 @@
+import type { Group } from '@code-pushup/models';
+import { AUDITS_MAP } from './constants.js';
+import {
+  filterAuditsByPluginConfig,
+  filterGroupsByOnlyAudits,
+} from './utils.js';
+
+describe('filterAuditsByPluginConfig', () => {
+  it('should return all audits when onlyAudits and skipAudits are not provided', () => {
+    const result = filterAuditsByPluginConfig({});
+    expect(result).toStrictEqual(Object.values(AUDITS_MAP));
+  });
+
+  it('should return all audits when onlyAudits is empty array and skipAudits is also empty array', () => {
+    const result = filterAuditsByPluginConfig({
+      onlyAudits: [],
+      skipAudits: [],
```

Comment:
There could also be a test for those params undefined (not set)

Reply:
Like providing an onlyAudits that does not actually exists? Because if you mean like not defining onlyAudits and skipAudits, it would be the test just above it with

```  
it('should return all audits when onlyAudits and skipAudits are not provided', () => {
    const result = filterAuditsByPluginConfig({});
    expect(result).toStrictEqual(Object.values(AUDITS_MAP));
  }); 
```

Reply:
Ah yeah! Then it is fine.